docs/eog/case-studies-cw20-event-roles.rst

```diff
@@ -0,0 +1,32 @@
+.. _CW20-Event-Roles: 
+
+CW20 Event Roles
+================
+
+It was clear that additional roles and duties were needed for an online event relative to an in-person event. The roles we needed included: 
```

Comment:
It read like the roles that follow are just needed for an online event - or is at least a bit ambiguous to me - whereas it's clear that e.g. a Chair and a Twitter person are needed for in person events also. - I think the Four sets ... and Zoom participants managers - were the unique ones for the online.